### PR TITLE
db14 examples

### PR DESCRIPTION
**Pull request by jamiltz**
@pasin ready for your review.

### CODE REVIEW

File: docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift [hunk 1]
```diff
@@ -175,10 +178,108 @@ class ViewController: UIViewController {
...
+        }
+    }
+    
+    // missing the count
+    // see https://developer.couchbase.com/documentation/server/4.6/n1ql/n1ql-language-reference/selectintro.html
+    func groupByQuery() {
+        let query = Query.select(
+                SelectResult.expression(Expression.property("country")),
+                SelectResult.expression(Expression.property("tz"))
+            )
```
Comment (jamiltz) [inline docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift]: not sure if DB14 supports `count` in the SELECT clause? N1QL equivalent query https://developer.couchbase.com/documentation/server/4.6/n1ql/n1ql-language-reference/selectintro.html#topic_xwp_r3k_dx__group-sort-limit

Comment (pasin) [inline docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift]: There is a Function.count(). I have never tried count(*) though.

Sample: https://github.com/couchbase/couchbase-lite-ios/blob/feature/2.0/Swift/Tests/QueryTest.swift#L378.

**Comment by pasin** [comment]
The PR looks good to me.

**Comment by jamiltz** [comment]
@pasin replaced the canned db with a bash script to save space and added a README. Can we merge this PR since those examples work? I'll open another PR with examples for query functions and collections.

**Pull request merged**